tools/autograd/derivatives.yaml

```diff
@@ -189,6 +198,9 @@
 - name: clone(Tensor self)
   self: grad
 
+- name: coalesce(Tensor self)
```

Comment:
@SsnL Hi Simon, can I ask if this backward means self and grad might have different nnz?